postman/xs2a gateway.postman_collection.json

```diff
@@ -75,6 +77,58 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "PIS/GetTransactionStatus/001a",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "26c04d43-e3bc-4bf0-acc8-59622c3b23aa",
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Transaction status is received\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.transactionStatus).to.eql(\"RCVD\");",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "X-Request-ID",
+						"value": "{{$guid}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "http://localhost:8081/v1/payments/sepa-credit-transfers/{{payment-id}}/status",
```

Comment:
seems could be parametrized

Reply:
I propose to do that in a separate PR (I could take care of it) because it will cause all the Postman scripts.

Reply:
Extracted host:port into an environment variable.